kedro-ppt-autogen-workflow/conf/base/catalog.yml

```diff
@@ -0,0 +1,44 @@
+# ============================================================================
+# DATA CATALOG - AutoGen PPT Generation Pipeline
+# ============================================================================
+
+# ==============================================================================
+# INPUT DATASETS
+# ==============================================================================
+
+instructions_yaml:
```

Comment:
Can you use the prompt datasets for this or do we need another prompt dataset for Autogen?

(This then makes me wonder if we could have a generic prompt dataset..)